src/amo/components/AddonAdminLinks/index.js

```diff
@@ -48,10 +48,14 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
       return null;
     }
 
-    const showCodeReviewLink = hasCodeReviewPermission && !isTheme(addon.type);
-    const showThemeReviewLink = hasThemeReviewPermission && isTheme(addon.type);
+    const isTheme = addon.type === ADDON_TYPE_STATIC_THEME;
+
+    const showCodeReviewLink =
+      hasCodeReviewPermission && !isTheme;
+    const showThemeReviewLink =
+      hasThemeReviewPermission && isTheme;
```

Comment:
Could you please reformat the lines? I am not sure running `yarn prettier-dev` will be enough.

Reply:
pls check